app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/prompts/DefaultBrowserPromptsExperimentMetrics.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2025 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.defaultbrowsing.prompts
+
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_DEFAULT_SET
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_DEFAULT_SET_VIA_CTA
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_STAGE_IMPRESSION
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_VALUE_DIALOG
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_VALUE_ENROLLED
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_VALUE_MENU
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_VALUE_STAGE_1
+import com.duckduckgo.app.browser.defaultbrowsing.prompts.DefaultBrowserPromptsExperimentMetrics.Companion.METRIC_VALUE_STAGE_2
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.ConversionWindow
+import com.duckduckgo.feature.toggles.api.MetricsPixel
+import com.duckduckgo.feature.toggles.api.MetricsPixelPlugin
+import com.squareup.anvil.annotations.ContributesMultibinding
+import dagger.SingleInstanceIn
+import javax.inject.Inject
+
+@ContributesMultibinding(AppScope::class)
+@SingleInstanceIn(AppScope::class)
+class DefaultBrowserPromptsExperimentMetrics @Inject constructor(
+    private val defaultBrowserPromptsFeatureToggles: DefaultBrowserPromptsFeatureToggles,
+) : MetricsPixelPlugin {
+    override suspend fun getMetrics(): List<MetricsPixel> {
+        return listOf(
+            MetricsPixel(
+                metric = METRIC_DEFAULT_SET,
+                value = METRIC_VALUE_ENROLLED,
```

Comment:
What's this for? Enrollment pixels are sent automatically when a cohort is assigned so you don't need to do this.

Reply:
This one will be removed, refs Thttps://app.asana.com/0/0/1208774988133227/1209263941236041/f. It was supposed to be a conversion pixel for day 0, before user has seen any prompts, but we decided that we don't need it.